<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWindow</class>
 <widget class="QMainWindow" name="RegWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>270</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>200</height>
    </size>
   </property>
   <widget class="QLineEdit" name="usernameEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit_1">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>70</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit_2">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>110</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请再输入一次密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regButton">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>150</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
